_drafts/2018-08-14-docviewer.md

```diff
@@ -0,0 +1,53 @@
+---
+
+title:  "The new and improved Docviewer"
+author: CsabaPeter
+image: "../img/posts/Docviewer_mockup.png"
+tags: [upgrade, UX, UI, design, research]
+
+---
+
+Sometimes there is a strong need to rethink an app from the ground up, because it has became too chunky and difficult to use with a lot of features that no one actually used. That's exactly what happened with sensenet's Docviewer, so as the newest addition to the product team as a UX designer my first task was to reimagine and redesign the app.
+In this article I'll guide you through the steps including user interviews, making wireframes to high fidelity prototypes, user testing, iterating and so on. 
+
+---
+
+## The first steps
+My work began with sitting down with the team and discussing what our main goal was (simplify the app and get rid of unused features). After that I started looking around for best practices like how other document viewer apps look like and what features they have. I used [Adobe Acrobat](https://acrobat.adobe.com/hu/hu/acrobat/pdf-reader.html), [Lumin](https://www.luminpdf.com/), and [Apple Preview](https://www.apple.com/lae/ios/ios-12-preview/) as starting point. 
+Since we wanted an app with mostly viewing capabilities and a little bit of editing qualities, Apple preview was the closest to our concept.
+
+After clarifying the main path that we should take, I conducted a few user interviews (six to be exact) to find out what people mainly looknig for when they open a document viewer. After the interviews were done and evaluated each interviewee's answers I began to make the wireframes. After several versions there was a clear winner among the team and from there the prototyping began.
```

Comment:
looknig --> looking

in this context, "among" means one of the team members was the winner.
Maybe: clear winner selected/chosen by the team...